src/components/app/content/cardList/index.js

```diff
@@ -1,18 +1,17 @@
 import React from 'react';
 import "./index.css"
+import WithLoadingDelay from "./WithLoadingDelay";
 import Card from "./card";
 
 function CardList(props) {
     return (
         <div className="cards">
-            {props.cards.map(card => {
-                return <Card cardData={card}
-                             saveCardData={props.saveCardData}
-                             readOnly={props.readOnly}
-                             checkedControl={props.checkedControl}
-                             checked={props.checkedCardIds.includes(card.id)}
-                             key={card.id}/>
-            })}
+            {props.cards.map(card => WithLoadingDelay(Card)({
```

Comment:
Not very suitable place to use component wrapping. Why don't you wrap Card component on export?